src/patternfly/components/FormControl/examples/FormControl.md

```diff
@@ -25,8 +25,6 @@ import './FormControl.css'
 <br><br>
 {{> form-control controlType="input" input="true" form-control--IsExpanded="true" form-control--attribute='type="text" value="Expanded" id="input-expanded" aria-label="Expanded input example"'}}
 <br><br>
-{{> form-control controlType="input" input="true" form-control--modifier="pf-m-search" form-control--attribute='type="search" value="Search" id="input-search" name="search-input" aria-label="Search input example"'}}
```

Comment:
Should we remove this from the docs, too? And mark the modifier to be removed in an upcoming breaking change?